biomage/experiment/info.py

```diff
@@ -0,0 +1,247 @@
+
+    header = list(content[0].keys())
+    table = []
+
+    for item in content:
+        table.append(list(item.values()))
+
+    print(tabulate(table, header, tablefmt="simple"))
+
+
+def _format_runs(content):
+    for run in content:
+        print(run['pipeline_type'].upper())
+        _print_tabbed('execution_arn', run['execution_arn'])
+
+        run_details = run['last_status_response'][run['pipeline_type']]
+
+        _print_tabbed('status\t', run_details['status'])
+        if(run_details['status'] != "SUCCEEDED"):
+
+            error_string = f"{run_details['error']['error']}"
+            if(run_details['error'].get('cause')):
+                error_string += run_details['error']['cause']
+
+            _print_tabbed('error\t', error_string)
+
+        _print_tabbed('start_date', run_details['startDate'])
+        _print_tabbed('stop_date', run_details['stopDate'])
+        _print_tabbed('completed_steps', run_details['completedSteps'])
+        print()
+
+
+def _pretty_print(result):
+
+    WIDTH = 30
+
+    print()
+    print("=" * WIDTH , "EXPERIMENT", "=" * WIDTH, "\n")
+    _format_item(result['info'])
+    print()
+
+    print("=" * WIDTH , "USERS", "=" * WIDTH, "\n")
+    _format_table(result['users'])
+    print()
+
+    print("=" * WIDTH , "SAMPLES", "=" * WIDTH, "\n")
+    if len(result['samples']):
+        _format_table(result['samples'])
+    else:
+        print("No samples uploaded")
+    print()
+
+    print("=" * WIDTH , "PIPELINE RUNNER", "=" * WIDTH, "\n")
+    _format_item(result['pipeline_runner'])
+    print()
+
+    print("=" * WIDTH , "RUNS", "=" * WIDTH, "\n")
+    if len(result['runs']):
+        _format_runs(result['runs'])
+    else:
+        print("Experiment has not been processed")
+
+
+@click.command()
+@click.option(
+    "-e",
+    "--experiment_id",
+    required=True,
+    help="Experiment ID to be copied.",
+)
+@click.option(
+    "-i",
+    "--input_env",
+    required=True,
+    default=STAGING,
+    show_default=True,
+    help="Input environment to pull the data from.",
+)
+@click.option(
+    "-p",
+    "--aws_profile",
+    required=False,
+    default=DEFAULT_AWS_PROFILE,
+    show_default=True,
+    help="The name of the profile stored in ~/.aws/credentials to use.",
+)
+@click.option(
+    "-o",
+    "--output",
+    required=False,
+    default='false',
+    show_default=True,
+    help="Output result to a format. Supported format: json, yaml",
+)
+def info(
+    experiment_id,
+    input_env,
+    aws_profile,
+    output
+):
+    """
+    Shows the required information related to the experiment.
+    It requires an open tunnel to the desired environment to fetch data from SQL:
+    `biomage rds tunnel -i production`
+
+    E.g.:
+    biomage experiment info -e 2093e95fd17372fb558b81b9142f230e -i production
+    """
+
+    db = init_db(SANDBOX_ID, USER, REGION, input_env, aws_profile)
+
+    info = _get_experiment_info(db, experiment_id)
+
+    pipeline_runner = {
```

Comment:
don't add all this custom logic. How we decide what to run will probably change soon making this outdated. Moreover, if you set the cpus/memory after running it on fargate it will report incorrect results. Just provide the current CPU, memory in the experiment

Reply:
ok